File: Interfaz/ProyectoFinal/src/app/Parametros/Salida/ListarTiposDeUsuarioOut.ts
import { TipoDeUsuario } from "src/app/Components/TipoDeUsuario/class/tipoDeUsuario";
import { ParametroBaseOut } from "./ParametroBaseOut";

export class ListarTiposDeUsuarioOut extends ParametroBaseOut{
    
    private tiposDeUsuario: TipoDeUsuario[] = [];
   
    constructor()
    {
        super();
    }

    public get TiposDeUsuario(): TipoDeUsuario[] {
        return this.tiposDeUsuario;
    }
    public set TiposDeUsuario(value: TipoDeUsuario[]) {
        this.tiposDeUsuario = value;
    }
    
}

File: ProyectoFinal/ProyectoFinal/Parametros/Salida/ListarTiposDeUsuarioOut.cs
using ProyectoFinal._2_Dominio.Entidades;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ProyectoFinal.Parametros.Salida
{
    public class ListarTiposDeUsuarioOut : ParametroBaseOut
    {
        public List<TipoDeUsuario> TiposDeUsuario { get; set; }
    }
}